magento/magento2
# Status of cache types changed after running setup:upgrade

### Description (*)
Fixed the issue that running `bin/magento setup:upgrade` re-enables layout, block_html and full_page caches.

### Fixed Issues (if relevant)
- Fixes #28186
- Fixes #29588

### Related Pull Requests

 NA

### Manual testing scenarios (*)

1. Disable cache types layout, block_html and full_page by running `bin/magento cache:disable block_html full_page layout`.
2. Run `bin/magento s:up`.
3. Enabling caches message displays in cli output and layout, block_html and full_page caches are enabled.
4. Expected result is those cache types should be disabled.

`Enabling caches:
Current status:
Array
(
    [layout] => 1
    [block_html] => 1
    [full_page] => 1
)`

### Questions or comments

NA

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29588: Status of cache types changed after running setup:upgrade
